modules/Movecraft/src/main/resources/localisation/movecraftlang_en.properties

```diff
@@ -51,6 +51,7 @@ Detection\ -\ Not\ Registered\ Pilot=Not one of the registered pilots on this cr
 Detection\ -\ Craft\ Type\ Cannot\ Be\ Named=Crafts of this type cannot be named
 Detection\ -\ ERROR\:\ Double\ chest\ found=Detection failed\! ERROR \: Double chest found
 Detection\ -\ Failed\ -\ Water\ contact\ required\ but\ not\ found=Detection failed\: Water contact required but not found
+Detection\ -\ Must\ Be\ Subcraft=Craft\ must\ be\ part\ of\ another\ craft
```

Comment:
We should review the non-uniform nature of detection warning, but this is fine for now